felix/rules/static.go

```diff
@@ -1040,45 +1040,18 @@ func (r *DefaultRuleRenderer) StaticRawTableChains(ipVersion uint8) []*Chain {
 }
 
 func (r *DefaultRuleRenderer) StaticBPFModeRawChains(ipVersion uint8,
-	wgEncryptHost, bypassHostConntrack, enforceRPF bool) []*Chain {
+	wgEncryptHost, bypassHostConntrack bool) []*Chain {
 
-	var rawRules, rpfRules []Rule
+	var rawRules []Rule
 
-	if enforceRPF {
+	if r.WireguardEnabled && len(r.WireguardInterfaceName) > 0 && wgEncryptHost {
```

Comment:
This was most likely incorrectly included with only the strict enforcement of RPF